govc/vm/customize.go

```diff
@@ -59,15 +60,11 @@ func (cmd *customize) Register(ctx context.Context, f *flag.FlagSet) {
 	f.StringVar(&cmd.domain, "domain", "", "Domain name")
 	f.StringVar(&cmd.host.Name, "name", "", "Host name")
 	f.Var(&cmd.mac, "mac", "MAC address")
-	cmd.mac = nil
```

Comment:
I think these need to stay, there are some apps embedding govc commands that call this func multiple times.
cc @gressd (#1869)